cli/util/upload_bin.go

```diff
@@ -32,12 +31,10 @@ import (
 )
 
 const (
-	// Number of objects to upload in a batch. All batches of uploadBatchSize size
-	// except the most recent one are guaranteed to be completed and don't contain gaps.
-	uploadBatchSize = 10000
-	// Number of objects to search in a batch. If it is larger than uploadBatchSize,
-	// it may lead to many duplicate uploads. We need to search with EQ filter to
-	// avoid partially-completed SEARCH responses.
+	// Number of objects to search in a batch. We need to search with EQ filter to
+	// avoid partially-completed SEARCH responses. If EQ search haven't found object
+	// we will have duplicates. We will have a risk of duplicates until #3645 is
+	// resolved (NeoFS guarantees search results).
```

Comment:
> haven't found

hasn't

s/we will have duplicates/the object will be uploaded one more time which may lead to duplicating objects